crates/nu-command/src/strings/str_/case/mod.rs

```diff
@@ -1,13 +1,13 @@
-pub mod camel_case;
-pub mod capitalize;
-pub mod downcase;
-pub mod kebab_case;
-pub mod pascal_case;
-pub mod screaming_snake_case;
-pub mod snake_case;
-pub mod str_;
-pub mod title_case;
-pub mod upcase;
```

Comment:
Good call. Somewhat surprising that this only pops piece by piece. (or was this one of those wildcard import warnings that seem to be raised with newer rust versions?)

Reply:
The warnings raised when using the latest rust `rustc 1.72.0-nightly (2d0aa5768 2023-06-18)`